packages/manager/tools/component-rollup-config/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Couldn't be inherit from a root configuration?

I mean creating a new package that export this configuration for instance.